modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/components/ReviewHeader/ReviewHeader.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+interface IProps {
+	description?: string;
+	name?: string;
+	title?: string;
+}
+
+const ReviewBod = ({description, name, title}: IProps) => (
```

Comment:
This component should be called ReviewHeader, isn't? 

```suggestion
const ReviewHeader = ({description, name, title}: IProps) => (
```

Taking the name of the file in consideration